CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# 0.3.1
+
+* DRM/KMS backend
+* Use `MAP_SHARED` on Redox; fixing behavior with latest Orbital
+* `OffscreenCanvas` support in web backend
+* Error instead of segfault on macOS if size isn't set
+* Make `fetch` error on Windows, where it wasn't working correctly
+* Implement `Error` trait for `SoftBufferError`
+* Dependency updates
```

Comment:
I think that some of these lines are useful as commit messages for us as developers, but as less useful as changes for users. For instance:

```
* DRM/KMS backend
```

Should probably be something more like:

```
* Add support for using `softbuffer` on raw TTYs, using the DRM/KMS interface.
```

Also, it would be nice if these entries were accompanied by PR numbers where applicable.